packages/perseus/src/widgets/expression/expression.stories.tsx

```diff
@@ -1,5 +1,6 @@
 import {KeypadContext} from "@khanacademy/keypad-context";
 import {action} from "@storybook/addon-actions";
+// biome-ignore lint/style/useImportType: <explanation>
```

Comment:
Are we using Biome now? I suspect this line shouldn't be here / isn't needed.

Reply:
No!  That was put there by accident. Thanks for catching!